### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by gspsp**
The URL is intentionally being removed.

### CODE REVIEW

**Comment by per1234** [comment]
Hello @gspsp. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by umbynos — APPROVED** [review]
Done:
```
2022/08/31 16:31:41 Loaded 5120 libraries from DB
Removing tasks
warning: Library Git clone folder /home/ubuntu/workspace/gitclones/github.com/gspsp/tasks not present
Success!
```

**Pull request merged**
